dev/release/release-tarball.sh

```diff
@@ -68,5 +71,9 @@ svn ci -m "Apache Arrow Rust ${version}" ${tmp_dir}/release
 echo "Clean up"
 rm -rf ${tmp_dir}
 
-echo "Success! The release is available here:"
+echo "Success!"
+echo "The release is available here:"
 echo "  https://dist.apache.org/repos/dist/release/arrow/${release_version}"
+
+echo "Clean up old versions from svn"
+(cd "${SOURCE_TOP_DIR}" && ./dev/release/remove-old-releases.sh)
```

Comment:
Why is the `cd` necessary, from what I can see the script is agnostic to the current working directory

Reply:
I don't have any good reason. Are you suggesting something like?

```suggestion
"${SOURCE_TOP_DIR}"/dev/release/remove-old-releases.sh
```